app/src/main/java/com/droidhats/campuscompass/views/MapFragment.kt

```diff
@@ -31,45 +29,25 @@ import com.droidhats.campuscompass.MainActivity
 import com.droidhats.campuscompass.R
 import com.droidhats.campuscompass.adapters.SearchAdapter
 import com.droidhats.campuscompass.helpers.Subject
-import com.droidhats.campuscompass.models.NavigationRoute
-import com.droidhats.campuscompass.models.Building
-import com.droidhats.campuscompass.models.GooglePlace
-import com.droidhats.campuscompass.models.IndoorLocation
-import com.droidhats.campuscompass.models.CalendarEvent
+import com.droidhats.campuscompass.models.*
```

Comment:
Was this intentional or was it done automatically by the IDE? If it wasn't intentional, I would fix these imports to be consistent through all classes